src/random_r.c

```diff
@@ -30,6 +30,7 @@
 
 #ifdef __MACH__
 # include "../include/features.h"
+#elif __unix__
```

Comment:
include literally nothing on unix :) apparently there's everything already there that's needed.